tests/php/test-fieldmanager-colorpicker-field.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Tests the Fieldmanager Colorpicker Field
+ *
+ * @group field
+ * @group colorpicker
+ */
+class Test_Fieldmanager_Colorpicker_Field extends WP_UnitTestCase {
+
+	protected $post;
+
+	public function setUp() {
+		parent::setUp();
+		Fieldmanager_Field::$debug = TRUE;
+
+		// insert a post
+		$this->post = $this->factory->post->create_and_get( array( 'post_title' => rand_str(), 'post_date' => '2009-07-01 00:00:00' ) );
+	}
+
+	public function test_basic_render() {
+		$fm = new Fieldmanager_Colorpicker( array( 'name' => 'test_colorpicker' ) );
+
+		ob_start();
+		$fm->add_meta_box( 'Test Colorpicker', 'post' )->render_meta_box( $this->post, array() );
+		$html = ob_get_clean();
+		$this->assertRegExp( '#<input class="[^"]*fm-colorpicker-popup[^>]+name="test_colorpicker"#', $html );
+	}
+
+	public function test_basic_save() {
+		$test_data = '#bada55';
```

Comment:
Hah! Incidentally, I got it from [core](https://make.wordpress.org/core/2012/11/30/new-color-picker-in-wp-3-5/), but @dannyb195 would certainly appreciate that.